### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by Manancode**
resolves issue #7083 . Fixing all broken links in the documentation

### CODE REVIEW

File: contributor_docs/friendly_error_system.md [hunk 1]
```diff
@@ -135,10 +135,10 @@ These functions are mainly responsible for catching errors and generating FES me
 
 For full reference, please see our [Dev Notes].
 
-[`_friendlyFileLoadError()`]: https://github.com/processing/p5.js/blob/main/contributor_docs/fes_reference_dev_notes.md#_friendlyfileloaderror
-[`_validateParameters()`]: https://github.com/processing/p5.js/blob/main/contributor_docs/fes_reference_dev_notes.md#validateparameters
-[`_fesErrorMontitor()`]: https://github.com/processing/p5.js/blob/main/contributor_docs/fes_reference_dev_notes.md#feserrormonitor
-[Dev Notes]: https://github.com/processing/p5.js/blob/main/contributor_docs/fes_reference_dev_notes.md
+[`_friendlyFileLoadError()`]: https://github.com/processing/p5.js/blob/daba49806a389e98cd63c62667928f61243c7fa1/contributor_docs/fes_contribution_guide.md#_friendlyerror
```
Comment (Qianqianye) [inline contributor_docs/friendly_error_system.md:138]: Hi, thanks for catching this. I think you can keep the 'main' in the URL, so the fixed the link would be like the following:
https://github.com/processing/p5.js/blob/main/contributor_docs/fes_contribution_guide.md#_friendlyerror

cc @limzykenneth

**Comment by limzykenneth** [comment]
I'd prefer not linking to a specific commit otherwise if we were to make updates to those documentation, these links will need to be updated again.

**Comment by Gautam-aman** [comment]
Can i also woek in this issue??

**Comment by limzykenneth** [comment]
@Gautam-aman #7117 will fix this so we don't need additional fixes for the linked issue for now. Do have a look at other open issues. Thanks.

**Comment by Qianqianye** [comment]
This is resolved in #7117. Thank you all!
@all-contributors please add @Manancode for documentation

**Comment by Qianqianye** [comment]
@all-contributors please add @Manancode for documentation

**Pull request closed without merging**